src/build.py

```diff
@@ -118,6 +119,9 @@ def fix_comments(str):
     elif sys.argv[cur_arg] == "--outro":
         cur_arg += 1
         outro_files = parse_files(sys.argv[cur_arg])
+    elif sys.argv[cur_arg] == "--output":
```

Comment:
There is no need for this. Python seems to think on your system, that it shall output CRLF line endings and not just LFs. This can be easily avoided by using `open( 'myfile', 'w', newline='' )`, so this should be the fix for `build.py` I believe :wink:.

Reply:
The issue was that the output was being written to stdout, and I wasn't sure how to prevent line endings from being changed while writing to stdout.

Reply:
What was happening was that CRLF was being output, and then Windows would see the trailing LF and, for some reason, change it to CRLF, resulting in 3 instances of CRCRLF in nuklear.h.

Reply:
I have resolved this issue by setting stdout to binary mode, reading files in binary mode, and replacing calls to print with stdout.write, as print appends LF.

Reply:
I think it's okay for this PR. Though I'm still curious how is it possible, that the `newline=''` solution didn't work (as it [should](https://docs.python.org/3/library/functions.html#open ) disable **any** translation and the sources all have LFs).